### PR TITLE
Update ax_multiobjective_nas_tutorial.py to use TensorboardMetric

### PR DESCRIPTION
**Pull request by mpolson64**
Ax's TensorboardCurveMetric is going away soon and is replaced by TensorboardMetric

## Description
My team (Adaptive Experimentation) wrote this tutorial a number of years ago. We have since updated the way our library [Ax](https://ax.dev/) interfaces with Tensorboard and are interested in reaping our old class TensorboardCurveMetric. In order to ensure this doesnt break our tutorial on the pytorch website we want to proactively update this tutorial to use our new class TensorboardMetric.

I tested this locally, but let me know if I can produce a more formal test plan.

cc @Balandat @dme65 

## Checklist

- [ ] ~~The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")~~
- [x] Only one issue is addressed in this pull request
- [ ] ~~Labels from the issue that this PR is fixing are added to this pull request~~
- [x] No unnecessary issues are included into this pull request.

### CODE REVIEW

**Review by Balandat — COMMENTED** [review]
What is the minimum Ax version with which this will work? May want to add this to the requirements so that this doesn't install an old version due to possible dependency version conflicts: https://github.com/pytorch/tutorials/blob/main/.ci/docker/requirements.txt#L33

File: .ci/docker/requirements.txt [hunk 1]
```diff
@@ -69,4 +69,4 @@ pygame==2.1.2
 pycocotools
 semilearn==0.3.2
 torchao==0.0.3
-segment_anything==1.0
+segment_anything==1.0
```
Comment (Balandat) [inline .ci/docker/requirements.txt:72]: should this be a newline here?

**Review by Balandat — APPROVED** [review]
Minor newline nit but otherwise lgtm, assuming build passes. 

@mpolson64 were you able to run the tutorial locally to validate that this works as intended?

**Review by svekars — APPROVED** [review]

**Comment by mpolson64** [comment]
> Minor newline nit but otherwise lgtm, assuming build passes.
> 
> @mpolson64 were you able to run the tutorial locally to validate that this works as intended?

Yup, ran totally fine on my conda env with minimal dependencies

**Pull request merged**